app/views/support/users/confirm_destroy.html.erb

```diff
@@ -0,0 +1,23 @@
+<%- title = t('page_titles.confirm_destroy_user') %>
+<%- content_for :before_content, govuk_link_to('Back', support_user_path(@user), class: 'govuk-back-link') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <span class="govuk-caption-xl"><%= @user.full_name %></span>
+      <%= title %>
+    </h1>
+
+    <p class="govuk-body">
+      Deleting this user will revoke their access to the service. If you need to reinstate them after deletion, contact the GHWT technical support team.
```

Comment:
Will it also remove them from the support portal (service now) and techsource?

Reply:
> Will it also remove them from the support portal (service now)

Yes, this was already done as part of https://github.com/DFE-Digital/get-help-with-tech/pull/748

Reply:
Minor bugfix on the back of your comment: https://github.com/DFE-Digital/get-help-with-tech/pull/994